packages/dendrite-pinecone/default.nix

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+{pkgs, ...}:
+with pkgs;
+  buildGo119Module rec {
+    pname = "dendrite-pinecone";
+    version = "0.9.1";
+
+    TcpPort = "49000";
+    McastUdpPort = "60606";
+    McastUdpIp = "239.0.0.114";
+    TcpPortInt = 49000;
+    McastUdpPortInt = 60606;
+    src = pkgs.fetchFromGitHub {
+      owner = "tiiuae";
+      repo = "dendrite";
+      rev = "feature/ghaf-integration";
```

Comment:
Please put commit hash here instead of branch name, and put the branch name into comment.

If you update that branch later, ghaf will start having build failures because of hash mismatches on clean ghaf builds, but it is not seen immediately because some machines have that hash stored in their /nix/stores.